.github/ISSUE_TEMPLATE/bug-fr.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bogue
+about: Générer un rapport de bogue pour nous aider à nous améliorer
+title: "[BUG] Nom de la fonctionnalité - Courte description du bogue"
+labels: ''
+assignees: ''
+
+---
+
+## Information nécessaire
+
+**Description du bogue**
```

Comment:
```suggestion
**Description du bogue**
<!-- Obligatoire -->
```

Reply:
Having it in the preceding section header is sufficient.